glass/rng.py

```diff
@@ -0,0 +1,132 @@
+"""Dispatcher functionality to unify JAX and NumPy RNG behavior."""
+
+from __future__ import annotations
+
+import math
+from threading import Lock
+from typing import TYPE_CHECKING, Any, TypeAlias
+
+import numpy as np
+from jax.dtypes import issubdtype, prng_key
+from jax.numpy import broadcast_shapes, shape
+from jax.random import (
+    key,
+    normal,
+    poisson,
+    split,
+    uniform,
+)
```

Comment:
Personally, I'd rather we `import jax.random` here. It's getting confusing with methods called `normal`, as well as direct imports called `normal`